keyboards/planck/keymaps/hvp1spc/config.h

```diff
@@ -0,0 +1,56 @@
+#define TAPPING_TERM 150
+#define IGNORE_MOD_TAP_INTERRUPT
+#define PERMISSIVE_HOLD
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x6060
+#define MANUFACTURER OLKB
+#define PRODUCT Planck
+#define DESCRIPTION A compact ortholinear keyboard
```

Comment:
This section isn't needed and should be removed.

Reply:
Removed layout altogether